timemachine/fe/lambda_schedule.py

```diff
@@ -19,6 +22,20 @@ def interpolate_pre_optimized_protocol(pre_optimized_protocol, num_windows):
     return lambda_schedule
 
 
+def bisect_lambda_schedule(
+    num_windows: int, lambda_interval: Optional[Tuple[float, float]] = None
+) -> NDArray[np.float64]:
+    """
+    Construct a lambda schedule with windows of size 2^N + 1 such that 2^N is the closest value to num_windows that is smaller.
```

Comment:
> Would it be fair to describe this as a sort of eager pre-computation?

Exactly.

Reply:
> Can't easily parse the first line of description
Will try to make it more interpretable. 

> the code doesn't appear to match the function name (function name: "bisect", implementation: "linspace"), but the implementation seems reasonable and useful.

I want the name to indicate that this should be used with bisection, as it is otherwise a naive lambda schedule. And how exactly the lambda schedule gets constructed should be independent (`np.linspace`, `greedy_bisection_step`) of its usage. At least that was how I thought about it.

Reply:
Renamed function to be `bisection_lambda_schedule`, which makes it sound more agnostic to the implementation: https://github.com/proteneer/timemachine/pull/1347/commits/c432836f2b3514f85dd10fbd6b3f35add44ab37f

Attempted to make the docstring clearer in https://github.com/proteneer/timemachine/pull/1347/commits/b971781149307aef05cd7f5b4e1701d44fdda551